pkg/ipam/crd.go

```diff
@@ -294,13 +294,9 @@ func (n *nodeStore) hasMinimumIPsInPool() (minimumReached bool, required, numAva
 
 	switch {
 	case n.ownNode.Spec.IPAM.MinAllocate != 0:
-		required = n.ownNode.Spec.IPAM.MinAllocate
-	case n.ownNode.Spec.ENI.MinAllocate != 0:
-		required = n.ownNode.Spec.ENI.MinAllocate
+		required = n.ownNode.Spec.IPAM.MinAllocate	
```

Comment:
This line has a trailing tab character at the end.

Reply:
thank you! will fix them